Learning_test_automation/Projekt#1/TeachersGradebook/TeachersRoom/GradeFile.cs

```diff
@@ -0,0 +1,78 @@
+﻿using SchoolData;
+
+namespace Infrastructure
+{
+    public class GradeFile
+    {
+        FileWizard fileWizard = new FileWizard();
+
+        public void GradeFileCreation(string createText)
+        {
+            string uniqName = "Grade";
+            fileWizard.SaveDataToFile(uniqName, createText);
+        }
+
+        public void AddGradeParametersFromFileToList()
+        {
+            string fileName = @"C:\Users\lemk\Desktop\TeachersGradebook\Grade.txt";
+
+            if (File.Exists(fileName))
+            {
+                using (StreamReader sr = new StreamReader(fileName))
+                {
+                    if (sr.Peek() >= 0)
+                    {
+                        string fileContent = sr.ReadToEnd();
+
+                        string[] grades = fileContent.Split('\n');
+
+                        SchoolSubjects schoolSubject = default;
+                        double gradeValue    = default;
+                        int studentId     = default;
+                        int gradeID = default;
+
+                        foreach (var lineWithGradeDate in grades)
```

Comment:
Logika od 34 do 62 linii to bardzo dobry kandydat na przynajmniej osobną metodę, a najlepiej nową klasę i nową metodę/nowe metody.

Reply:
Narazie zamienie na osobną metodę a potem zobaczymy.